style.css

```diff
@@ -1,3 +1,10 @@
 body {
-	
-}
\ No newline at end of file
+  background-color: grey;
+  font-family: "GothamPro";	
+}
+ 
+ .item__price {
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/step/pullRequest?prid=5896505)